package.json

```diff
@@ -39,24 +39,24 @@
   "homepage": "https://github.com/electron/electron-api-demos#readme",
   "devDependencies": {
     "chai": "^3.4.1",
-    "chai-as-promised": "^5.1.0",
+    "chai-as-promised": "^6.0.0",
     "devtron": "^1.3.0",
     "electron": "^1.3.5",
-    "electron-packager": "^7.0.1",
-    "electron-windows-store": "^0.3.0",
+    "electron-packager": "^8.0.0",
+    "electron-windows-store": "^0.5.2",
     "electron-winstaller": "^2.2.0",
-    "mocha": "^2.3.4",
+    "mocha": "^3.1.0",
     "request": "^2.70.0",
     "rimraf": "^2.5.2",
-    "signcode": "^0.4.0",
+    "signcode": "^0.5.0",
     "snazzy": "^5.0.0",
-    "spectron": "~3.2.6",
-    "standard": "^6.0.8"
+    "spectron": "^3.4.0",
```

Comment:
The minor version of spectron tracks the minor version of Electron, so 3.4 targets electron 1.4, and the `~` should be used instead of the `^` to enforce this.